src/components/editions/headerMedia.tsx

```diff
@@ -115,6 +129,19 @@ const HeaderImage: FC<Props> = ({ item }) => {
 					<StarRating item={item} />
 				</figure>
 			);
+		} else if (media.video.atomId !== undefined) {
```

Comment:
Could we use `MainMediaKind.video` here?

Reply:
We tried evaulating `MainMediaKind.video && media.video.atomId` , but we was getting an error to say we were doing an unneccessary condition check, it's because `MainMediaKind` only has `Image` or `Video` and we are already checking for Image above so it can only be a Video.

Reply:
ah ok, in that case can it just be an `else`?

Reply:
I believe `atomId` could be undefined even if we have MainMediaKind.video (following a capi type)

Reply:
Hmm not sure if it can be, because `atomId` is optional, `else` wouldn't handle if `atomId` is missing?

Reply:
ah ok, so you can have a video but no `atomId`?

In that case stick with what you have 👍

Reply:
Although looking at it again after [this line](https://github.com/guardian/apps-rendering/blob/25a0994502d587310b8e17c8b3c88fbc6dfbb03f/src/video.ts#L19) we can pretty much depend on atomId always being there, atomId doesn't really need to be optional in the Video interface. What do you think @frankie297? shall we update the type?

Reply:
Yeh I think you are right, lets update the type